src/virt.c

```diff
@@ -578,15 +613,23 @@ lv_read (void)
                     &stats, sizeof stats) != 0)
             continue;
 
+	char *new_path=NULL;
```

Comment:
What makes this path "new"? I suggest to rename this variable to `type_instance` to make its purpose clear.